### ISSUE TITLE
Invalid serialization of tuple enum variants

### ISSUE THREAD

**Issue by dunnock** [simd-lite/simd-json#395]
# Version

simd-json v0.14.0

# Description

When writing tuple enum variants simd_json is missing to close block bracket

 # Test

When trying to run following test

```
#[test]
fn write_err_simd_json_tuple() {
    #[derive(Serialize, Clone)]
    enum ErrType {
        Instruction(i64, i64),
    }
    let err = ErrType::Instruction(2, 68800);
    #[derive(Serialize)]
    struct Status {
        err: Option<ErrType>,
    }

    assert_eq!(
        simd_json::to_string(&Status {
            err: Some(err.clone())
        })
        .unwrap(),
        serde_json::to_string(&Status { err: Some(err) }).unwrap(),
    );
}

```

# Error

It results with error:
```
---- messages::tests::write_err_simd_json_tuple stdout ----
thread 'messages::tests::write_err_simd_json_tuple' panicked at src/messages.rs:134:9:
assertion `left == right` failed
  left: "{\"err\":{\"Instruction\":[2,68800}}"
 right: "{\"err\":{\"Instruction\":[2,68800]}}"
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```

**Comment by Licenser** [comment]
Good catch, I'll fix and publish a new version this evening

**Comment by Licenser** [comment]
@dunnock released as 0.14.1, tanks again :)

**Comment by dunnock** [comment]
Awesome, thank you for prompt fix!
